osu.Game/Overlays/Chat/ChatReportReason.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.ComponentModel;
+using osu.Framework.Localisation;
+using osu.Game.Resources.Localisation.Web;
+
+namespace osu.Game.Overlays.Chat
+{
+    public enum ChatReportReason
+    {
+        [Description("Insulting People")]
+        [LocalisableDescription(typeof(UsersStrings), nameof(UsersStrings.ReportOptionsInsults))]
+        Insults,
+
+        [Description("Spam")]
+        [LocalisableDescription(typeof(UsersStrings), nameof(UsersStrings.ReportOptionsSpam))]
+        Spam,
+
+        [Description("Cheating")]
+        [LocalisableDescription(typeof(UsersStrings), nameof(UsersStrings.ReportOptionsCheating))]
+        FoulPlay,
```

Comment:
This is not a valid chat report reason. See: https://github.com/ppy/osu-web/blob/master/app/Models/UserReport.php#L49, https://github.com/ppy/osu-web/blob/master/app/Models/UserReport.php#L39.

I'll fix it myself, but @cdwcgt please pay attention to this sort of thing in the future if you're going to be implementing osu-web requests yourself.

Reply:
This is something that didn't have before I implemented this function. My list of reasons comes from stable.
Does this place need to be inconsistent with stable?

Reply:
It needs to be consistent with web. End of story.

Reply:
ok